addons/languages/nias/apk/src/main/play/release-notes/en-US/alpha.txt

```diff
@@ -1 +1,12 @@
-* New owner
+1. Improved the nias wiki layout:
```

Comment:
The file can not be more than 500 characters long (this is a limitation from Google Play).
From the error message: `whatsnew file can not be longer than 500 characters! Currently 598`

Reply:
Sorry, I didn't know about this. I'll update the relevant files and submit the pull request later.